Docs/source/running_cpp/parameters.rst

```diff
@@ -1216,10 +1216,21 @@ Diagnostics and output
 * ``<reduced_diags_name>.extension`` (`string`) optional (default `txt`)
     The extension of the output file.
 
-* ``<reduced_diags_name>.separator`` (`string`) optional (default `,`)
+* ``<reduced_diags_name>.separator`` (`string`) optional (default a `whitespace`)
     The separator between row values in the output file.
-    The default separator is comma, i.e. the output file is in
-    the CSV (comma separated value) format.
+    The default separator is a whitespace.
+    If one uses a python script to read the data,
+    one example is the following:
+
+    ::
+
+        import numpy
+        data = numpy.genfromtxt("filename.txt")
+        header = numpy.genfromtxt("filename.txt",comments="@",max_rows=1,dtype="str")
```

Comment:
```suggestion
        header = numpy.genfromtxt("filename.txt",comments="@",max_rows=1,dtype="str")
        data_dict = {key: data[i,:] for i, key in enumerate(header)}
        # And then you may access the values for a given field, e.g., time, with
        data_dict['time']
```
So be it, but a dictionary is, among other things, much more stable, because if you add some output or change the order of columns, it still works.
BTW could you check the few lines I added are correct?

Reply:
One thing is that the strings in the `header` array are actually:
```
header[0] = "#[1]step"
header[1] = "[2]time(s)"
...
```
I feel like it has to be `data_dict['[2]time(s)']`, is that true?
That's why I didn't talk too much about it :)
I was thinking to let the user figure out fancy ways to get rid of '#[1]`, etc.